types/normalize-port/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        // This package uses the Function type, and it will take effort to fix.
+        "ban-types": false
```

Comment:
This doesn't appear to use the `Function` type. This file should just be `{ "extends": "dtslint/dt.json" }`.